app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -233,48 +201,37 @@ class Exposure extends React.Component {
     });
   };
 
-  updateSOandNRSValidations = patient => {
-    if (!patient.symptom_onset && !patient.no_reported_symptoms) {
+  updateSOandPosLabValidations = (patient, first_positive_lab) => {
```

Comment:
I don't love this method name, wondering if we could just call it `updateAsymptomaticValidations` or something a hair cleaner

Reply:
It was named this way to keep it consistent with the naming convention of `updateLDEandCEValidations`. I don't like the idea of including the word "asymptomatic" in it because we're trying to steer away from that terminology with the latest requirements/updates of this feature. How about `updateExposureValidations` and `updateIsolationValidations`?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-238f84238ca8a6fee3e8ebb55563658dd6716d2cf063b2bcf655b1b37de161fcR204